### ISSUE TITLE
Is there a way to make a tree chart populate vertically rather than horizontally? 

### ISSUE THREAD

**Issue by abilashs90** [andreaferretti/paths-js#40]
Hi, Is there a way to make a tree chart populate vertically rather than horizontally?

**Comment by andreaferretti** [comment]
Well, what I usually do is insert the whole tree inside a `g` element with the `transform=rotate(90)` attribute.

This has the unfortunate effect of rotating labels as well, but you can insert the opposite rotation.

I have been thinking whether to add a flag to make the tree vertical, but I am not sure if the additional parameter would be worth, given that SVG graphics allows rotations

**Comment by abilashs90** [comment]
Yes, I tried rotate, but that needs a lot of changes wrt margins etc. It would be a nice addon to a great library.
